src/main/java/com/bobocode/bibernate/QueryHelper.java

```diff
@@ -7,18 +7,35 @@
 import java.util.function.Consumer;
 import java.util.function.Function;
 
+/**
+ * Util class that provides methods for performing logic within session and transaction scope
+ */
 public class QueryHelper {
 
     private QueryHelper() {
     }
 
+    /**
+     * Creates a new session using the provided session factory and performs the passed action within the session scope.
```

Comment:
(nit) would be good to add link to the SessionFactory using {@link SessionFactory}

Reply:
the same for Session